recipe/build.bat

```diff
@@ -4,7 +4,16 @@ if "%1"=="cli" (set cli=1 && set build=build_cli) else (set cli=0)
 
 if "%1"=="gui" (set gui=1 && set build=build_gui) else (set gui=0)
 
-echo cli is %cli% and gui is %gui%
+if "%cuda_compiler_version%" == "None" (set gpu=0) else (set gpu=1)
+
+echo cli is %cli%, gui is %gui% and gpu is %gpu%
+
+echo CUDA_HOME is %CUDA_HOME%
```

Comment:
It seems that `CUDA_HOME` is not set correctly even if it should be set by nvcc, if I understand correctly?! @jaimergp would you have any idea about this?

Reply:
Try CUDA_PATH in Windows (although we should be setting that too).